### DISCUSSION TITLE
Placeholder for setting in editor

### DISCUSSION THREAD

**Discussion by silvar23** [sulu/sulu#5946] [Q&A] [answered]
Reading the sulu docs but can't find anything about my issue.

I like to use placeholders for page independent settings inside the editor. 

In editor the text is:
Our company %setting.company_name% is great.

In output will be replaced with snippet, example result:
Our company XYZ GmbH is great.

Or should I just build my own twig filter for this?

Thank you!

**Comment by niklasnatter** [comment, marked as answer]
Hey,
thanks for reaching out! 

Sulu does not provide any built-in functionality for this as far as I know. At the moment, I would recommend to implement a Twig extension/filter in your project and document the supported placeholders using a [`info_text` in your template](https://docs.sulu.io/en/2.2/book/templates.html#help-text).

**Reply by silvar23** [reply]
Thank you!
